validator/src/main.rs

```diff
@@ -1446,12 +1446,21 @@ pub fn main() {
             ..RuntimeConfig::default()
         },
         staked_nodes_overrides: staked_nodes_overrides.clone(),
-        replay_slots_concurrently: matches.is_present("replay_slots_concurrently"),
         use_snapshot_archives_at_startup: value_t_or_exit!(
             matches,
             use_snapshot_archives_at_startup::cli::NAME,
             UseSnapshotArchivesAtStartup
         ),
+        replay_forks_threads: if matches.is_present("replay_slots_concurrently") {
```

Comment:
probably worth making the arg name strings constants to avoid typo errors

Reply:
Done in https://github.com/anza-xyz/agave/pull/236/commits/567a07e81986e3d9e1065080b411861dbf218e90